iOSClient/SceneDelegate.swift

```diff
@@ -178,6 +178,9 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
 
     func sceneDidEnterBackground(_ scene: UIScene) {
         NextcloudKit.shared.nkCommonInstance.writeLog("[INFO] Scene did enter in background")
+        ///
+        database.backupTableAccountToFile()
+        ///
```

Comment:
Documentation comments are syntactically inappropriate here.
```suggestion
        database.backupTableAccountToFile()
```